src/mobile/src/ui/views/onboarding/SeedReentry.js

```diff
@@ -87,7 +87,7 @@ class SeedReentry extends Component {
     constructor() {
         super();
         this.state = {
-            reenteredSeed: null,
+            reenteredSeed: '',
```

Comment:
I think this change might break QR code scanning. On line 158 we set `reenteredSeed` to a trit value:

```js
this.setState({ reenteredSeed: trytesToTrits(data) });
```
Since `trytesToTrits` returns an `Int8Array` I don't think we can initialize it to an empty string

Reply:
It doesn't break it